tests/containers/skopeo.pm

```diff
@@ -19,10 +19,10 @@ use containers::common qw(install_packages);
 my $workdir = '/tmp/test';
 
 sub run {
-    my ($self, $args) = @_;
-
     # Required packages
     my @packages = qw(skopeo jq);
+    my ($self, $args) = @_;
+    my $runtime = $args->{runtime};
```

Comment:
I would make this variable global so it can be reused by the `cleanup()` function.

Reply:
tbh this is the first thing that I tried and it did not work - the cleanup function failed because `$runtime` was replaced with an empty string. I am not sure why :\